documentation/user_doc.md

```diff
@@ -0,0 +1,55 @@
+# Nexus Replication Plugin
+
+[https://github.com/griddynamics/nexus-replication-plugin](https://github.com/griddynamics/nexus-replication-plugin)
+
+This is Nexus plugin that provides replication logic when new (Maven) artifacts are stored in a hosted repository. 
+Basically it POST an HTTP requests from master to peers instance when new artifact is saved then peers invoke artifact pooling into proxy repository.
+
+## Setting up plugin
+
+Plugin should be installed into master and peer Nexus servers both.
+
+### Master instance
+
+1. Download source from [https://github.com/griddynamics/nexus-replication-plugin](https://github.com/griddynamics/nexus-replication-plugin).
+2. Run `mvm package` in root directory (project requires Apache Maven 3.0.4 -- 3.1 and JDK 1.7+).
```

Comment:
mv**m** -> mv**n**

Maven version - anything below 3.2.1 is considered outdated nowadays. It should work on Maven 3.2.1 and higher.